lib/concerns/music.rb

```diff
@@ -0,0 +1,22 @@
+module Concerns
+  module MusicBase
+    def all
+      class_variable_get :@@all
+    end
+
+    def destroy_all
+      all.clear
+    end
+
+    def create(name)
+      new(name).save
+    end
+  end
+
```

Comment:
Trainer's Feedback: Move the instance module to a different file so that class module are separate/distinct from instance modules.